edgar-filings-examples/README.md

```diff
@@ -0,0 +1,72 @@
+# Edgar Filings Examples
+
+This is a collection of examples of how to use the Algoseek's Edgar Filing Dataset.
+
+## Table of Contents
+
+- [Edgar Filings Examples](#edgar-filings-examples)
+  - [Table of Contents](#table-of-contents)
+  - [Available notebooks](#available-notebooks)
+    - [Financial Filings](#financial-filings)
+    - [Shares Outstanding](#shares-outstanding)
+  - [Prerequisites](#prerequisites)
+    - [Database credentials](#database-credentials)
+    - [Software Requirements](#software-requirements)
+  - [Usage](#usage)
+  - [Alternative usage](#alternative-usage)
+
+## Available notebooks
+
+The following notebooks are available in this repository:
+
+### Financial Filings
+
+This notebook shows how to get the financial filings of a company as reported in the 10-K and 10-Q filings.
+
+### Shares Outstanding
+
+This notebook shows how to get the shares outstanding of a company as reported in the 10-K and 10-Q filings.
+
+## Prerequisites
+
+### Database credentials
+
+For the examples to work, you will need to have credentials to access our database service, if you don't have them please feel free to contact your account executive and we'l provide it to you.

```

Comment:
Maybe better to use an account manager.
we`l to we`ll

Reply:
Will update if the access method to Arda changes.